src/index.ejs

```diff
@@ -85,7 +85,9 @@
           onComplete: function() {
             /*callback for when */ console.log("everything is complete")
           },
-          steps: steps
+          steps: steps,
+          locale: 'es',
+          customStrings: {new_string: 'This is my custom string'}
```

Comment:
it looks like this will be a breaking change. don't think we will be able to preserve the existing partial copy customization

Reply:
Not necessarily. We could still allow partial customisation, like in the welcome screen. We could have two separate components, one with localised strings and one that allows for customisable parts.